react-components/src/components/Architecture/pointsOfInterest/PoiInfoSection.tsx

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { Flex, Input, TextLabel } from '@cognite/cogs.js';
+import { type ReactNode } from 'react';
+import { useTranslation } from '../../i18n/I18n';
+import { useSelectedPoi } from './useSelectedPoi';
+import styled from 'styled-components';
+
+export const PoiInfoSection = (): ReactNode => {
+  const { t } = useTranslation();
+  const selectedPoi = useSelectedPoi();
+  if (selectedPoi === undefined) {
+    return undefined;
+  }
+  const { name, description } = selectedPoi.properties;
+  return (
+    <InfoSectionContainer direction="column" justifyContent="space-between" gap={8}>
+      <Flex direction="column">
+        <TextLabel text={t({ key: 'TITLE' })} />
+        <Input placeholder={t({ key: 'TITLE_PLACEHOLDER' })} value={name} fullWidth />
+      </Flex>
+      <Flex direction="column">
+        <TextLabel text={t({ key: 'DESCRIPTION' })} />
+        <Input placeholder={t({ key: 'DESCRIPTION_PLACEHOLDER' })} value={description} fullWidth />
```

Comment:
Do these need translations? We have "NAME", so maybe we can use that instead of "TITLE". We also have a POI-specific description placeholder. The rest, I'm not sure :/

Reply:
On second thought - I don't think we need placeholders here? They should never be empty, and besides, the label above the input field should be enough